### PR TITLE
New Branch

### PR DESCRIPTION
**Pull request by ZhanpZ**
MUI

### CODE REVIEW

File: app/src/pages/index.tsx [hunk 1]
```diff
@@ -15,6 +16,7 @@ export default function Home() {
                     <Stack gap={3}>
                         <PageHeader>CL311 Home Page</PageHeader>
                         <p>Welcome to CodeDay Labs 311!</p>
+                        <DeleteIcon/><DeleteIcon color="primary" /><DeleteIcon color="secondary" /><DeleteIcon color="success" /><DeleteIcon color="action" /><DeleteIcon color="disabled" />
```
Comment (MinieMoe) [inline app/src/pages/index.tsx:19]: maybe put the code in separate lines for readability. other than that, it looks good!

**Review by MinieMoe — APPROVED** [review]
beside the small comment I made, it looks good to me!

File: app/src/pages/index.tsx (hunk 1 above)
Comment (andreybutenko) [reply app/src/pages/index.tsx:19]: I agree :) I think this is okay for testing code so I'll approve too

**Review by andreybutenko — APPROVED** [review]
Nice job! In the future, could you please use a more descriptive title for your PRs? Thanks :D

**Pull request merged**